app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/Tab.kt

```diff
@@ -0,0 +1,443 @@
+import com.hedvig.android.design.system.hedvig.TabDefaults.TabSize.Mini
+import com.hedvig.android.design.system.hedvig.TabDefaults.TabStyle
+import com.hedvig.android.design.system.hedvig.TabDefaults.TabStyle.Default
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.ButtonSecondaryAltResting
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.ButtonSecondaryResting
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.SurfacePrimary
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.TextPrimary
+import com.hedvig.android.design.system.hedvig.tokens.LargeTabTokens
+import com.hedvig.android.design.system.hedvig.tokens.MediumTabTokens
+import com.hedvig.android.design.system.hedvig.tokens.MiniTabTokens
+import com.hedvig.android.design.system.hedvig.tokens.SmallTabTokens
+import kotlin.math.ceil
+
+// only for up to 6 TabItems!
+@Composable
+fun HedvigTabRowMaxSixTabs(
+  tabTitles: List<String>,
+  selectedTabIndex: Int,
+  onTabChosen: (index: Int) -> Unit,
+  modifier: Modifier = Modifier,
+  tabSize: TabSize = TabDefaults.defaultSize,
+  tabStyle: TabStyle = TabDefaults.defaultStyle,
+) {
+  var currentIndicatorOffset by remember { mutableStateOf(IntOffset(0, 0)) }
+  val indicatorOffset: IntOffset by animateIntOffsetAsState(
+    targetValue = currentIndicatorOffset,
+    animationSpec = tween(
+      durationMillis = 600,
+      easing = FastOutSlowInEasing,
+    ),
+    label = "",
+  )
+  var oneLineHeight by remember { mutableIntStateOf(0) }
+  Box(
+    modifier = modifier
+      .clip(tabSize.rowShape)
+      .background(tabStyle.colors.tabRowBackground)
+      .height(intrinsicSize = IntrinsicSize.Min)
+      .padding(tabSize.rowPadding),
+  ) {
+    Box(Modifier.withoutPlacement()) {
+      // to get one line height
+      HedvigText(
+        text = "measurement",
+        style = tabSize.textStyle,
+        modifier = Modifier
+          .onSizeChanged {
+            oneLineHeight = it.height
+          }
+          .padding(tabSize.tabPadding),
+      )
+    }
+    Layout(
+      contents = listOf(
+        {
+          TabIndicator(
+            indicatorColor = tabStyle.colors.chosenTabBackground,
+            indicatorShape = tabSize.tabShape,
+          )
+        },
+        {
+          tabTitles.forEachIndexed { index, title ->
+            TabItem(
+              modifier = Modifier
+                .clip(tabSize.tabShape),
+              onClick = {
+                onTabChosen(index)
+              },
+              text = title,
+              textStyle = tabSize.textStyle,
+              tabTextColor = tabStyle.colors.textColor,
+              contentPadding = tabSize.tabPadding,
+            )
+          }
+        },
+      ),
+    ) { allMeasurables: List<List<Measurable>>, constraints ->
+      val tabItemsMeasurables = allMeasurables[1]
+      if (tabItemsMeasurables.size <= 1) {
```

Comment:
Can do `isEmpty()` I think.
```suggestion
      if (tabItemsMeasurables.size.isEmpty()) {
```
Not important at all, just a bit easier on the eye when looking at this without having to look at the numbers

Reply:
I'd like to not layout anything if size==1 too:)

Reply:
Ah, right! A tab bar with one option certainly would be quite silly wouldn't it 😅